src/engraving/libmscore/textbase.cpp

```diff
@@ -1668,10 +1668,10 @@ TextBase::TextBase(Score* s, ElementFlags f)
 TextBase::TextBase(const TextBase& st)
     : Element(st)
 {
-    _cursor                      = st._cursor;
+    _cursor                      = new TextCursor(this);
```

Comment:
Fixes a critical bug where the TextCursor._text pointer can be left pointed to deleted memory (happens now when copying text elements)